source/examples.rst

```diff
@@ -0,0 +1,157 @@
+********
+Examples
+********
+
+Introduction
+============
+
+The model and observation data are located at NERSC, so you can run the examples on Cori or Edison.
+
+First, make sure you're using version 2.0.0 or greater of e3sm_diags.
+
+Then do one of the following:
+
+* Enter a ``conda`` environment that has ``e3sm_diags`` installed.
+   * Either follow `a quickstart guide <quickguides/index.html>`__ or `the instructions here <install.html>`__.
+* If you have the container downloaded, then download `this script <https://raw.githubusercontent.com/E3SM-Project/e3sm_diags/master/acme_diags/container/e3sm_diags_container.py>`__ to run it.
+   * If you're using a container, replace every instance of
+     ``e3sm_diags`` in this guide with ``python e3sm_diags_container.py --<your_container_runtime>``.
+   * See `this guide <quickguides/quick-guide-cori.html>`__ for more information.
```

Comment:
This seems like it should instead be a link to setting up the container. It currently links to the Cori quick guide. Is there a better link to use or is this what we want here?

Reply:
Since we are not supporting containers currently. Let's point to the installation guide, instead?

Reply:
In that case, should I just remove the part about containers entirely?

Reply:
Yes!